<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabUi</class>
 <widget class="QWidget" name="TabUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="homepageButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 black);
image: url(:/Resources/HomeIcon.png);
image-position: 1,1;
font-size: 18px;

border:grey;
border-bottom: 1px solid white;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
border-bottom: 1px solid white;

}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
border-bottom: 1px solid white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>520</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 black);

QWidget::move(40, 2);
border-bottom: 1px solid white;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>                                            Calgary Solar Car Team</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="Digital_Clock">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ToolTipText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ToolTipText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>100</red>
               <green>100</green>
               <blue>100</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ToolTipText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(100,100,100), stop: 1 black);
border-bottom: 1px solid white;


</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="controlButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

color: rgb(255,255,255);

background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
   
                                 stop: 0 rgb(60,60,60), stop: 1 rgb(60,60,60));

image: url(:/Resources/SteeringIcon.png);

image-position: 1,1;

font-size: 18px;

border:grey;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
        </property>
        <property name="text">
         <string> Control</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="batteryButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(60,60,60), stop: 1 rgb(60,60,60));
image: url(:/Resources/BatteryIcon.png);
image-position: 1,1;
font-size: 18px;

border: grey;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
        </property>
        <property name="text">
         <string>Battery</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="motorButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(60,60,60), stop: 1 rgb(60,60,60));
image: url(:/Resources/EngineIcon.png);
image-position: 1,1;
font-size: 18px;

border:grey;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
        </property>
        <property name="text">
         <string>Motor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="faultButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>134</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{

color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(60,60,60), stop: 1 rgb(60,60,60));
image: url(:/Resources/GearIcon.png);
image-position: 1,1;
font-size:18px;

border:grey;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
        </property>
        <property name="text">
         <string>Faults</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mpptButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(60,60,60), stop: 1 rgb(60,60,60));
image: url(:/Resources/MPPTIcon.png);
image-position: 1,1;
font-size: 18px;

border:grey;
}
QPushButton:hover{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(160,160,160), stop: 1 rgb(120,120,120));
}
QPushButton:pressed{
color: rgb(255,255,255);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(120,120,120), stop: 1 rgb(80,80,80));
}</string>
        </property>
        <property name="text">
         <string>MPPT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
